editor/editor_command_palette.cpp

```diff
@@ -81,60 +85,55 @@ void EditorCommandPalette::_update_command_search(const String &search_text) {
 
-	if (entries.size() > 0) {
-		if (!search_text.is_empty()) {
-			SortArray<CommandEntry, CommandEntryComparator> sorter;
-			sorter.sort(entries.ptrw(), entries.size());
-		}
+	if (entries.is_empty()) {
+		get_ok_button()->set_disabled(true);
 
-		const int entry_limit = MIN(entries.size(), 300);
-		for (int i = 0; i < entry_limit; i++) {
-			String section_name = entries[i].key_name.get_slice("/", 0);
-			TreeItem *section;
+		return;
+	}
 
-			if (sections.has(section_name)) {
-				section = sections[section_name];
-			} else {
-				section = search_options->create_item(root);
+	if (!search_text.is_empty()) {
+		SortArray<CommandEntry, CommandEntryComparator> sorter;
+		sorter.sort(entries.ptrw(), entries.size());
+	}
 
-				if (!first_section) {
-					first_section = section;
-				}
+	const int entry_limit = MIN(entries.size(), 300);
+	for (int i = 0; i < entry_limit; i++) {
+		String section_name = entries[i].key_name.get_slice("/", 0);
+		TreeItem *section;
 
-				String item_name = section_name.capitalize();
-				section->set_text(0, item_name);
+		if (sections.has(section_name)) {
+			section = sections[section_name];
+		} else {
+			section = search_options->create_item(root);
 
-				sections[section_name] = section;
-				section->set_custom_bg_color(0, search_options->get_theme_color("prop_subsection", "Editor"));
-				section->set_custom_bg_color(1, search_options->get_theme_color("prop_subsection", "Editor"));
+			if (!first_section) {
+				first_section = section;
 			}
 
-			TreeItem *ti = search_options->create_item(section);
-			String shortcut_text = entries[i].shortcut_text == "None" ? "" : entries[i].shortcut_text;
-			ti->set_text(0, entries[i].display_name);
-			ti->set_metadata(0, entries[i].key_name);
-			ti->set_text_align(1, TreeItem::TextAlign::ALIGN_RIGHT);
-			ti->set_text(1, shortcut_text);
-			Color c = Color(1, 1, 1, 0.5);
-			ti->set_custom_color(1, c);
-		}
-
-		TreeItem *to_select = first_section->get_first_child();
-		to_select->select(0);
-		to_select->set_as_cursor(0);
-		search_options->scroll_to_item(to_select);
+			String item_name = section_name.capitalize();
+			section->set_text(0, item_name);
+			section->set_selectable(0, false);
+			section->set_selectable(1, false);
+			section->set_custom_bg_color(0, search_options->get_theme_color("prop_subsection", "Editor"));
+			section->set_custom_bg_color(1, search_options->get_theme_color("prop_subsection", "Editor"));
 
-		get_ok_button()->set_disabled(false);
-	} else {
-		TreeItem *ti = search_options->create_item(root);
-		ti->set_text(0, TTR("No matching commands found"));
-		ti->set_metadata(0, "");
-		Color c = Color(0.5, 0.5, 0.5, 0.5);
-		ti->set_custom_color(0, c);
-		search_options->deselect_all();
+			sections[section_name] = section;
+		}
 
-		get_ok_button()->set_disabled(true);
+		TreeItem *ti = search_options->create_item(section);
+		String shortcut_text = entries[i].shortcut_text == "None" ? "" : entries[i].shortcut_text;
```

Comment:
Should this be TTR("None")? Not 100% sure but I think the shortcut name could be the translated version here? And I think None is translated in shortcut? On mobile atm, I will check later 😊

Reply:
I'm honestly having trouble finding where that could even be set as "None".

Reply:
I think it is here

https://github.com/godotengine/godot/blob/31dfdcb69eb4d98a399243f5fc93b8e14b2e808a/editor/editor_command_palette.cpp#L141

But indeed it does not appear to be translated.

Reply:
Also here:

https://github.com/godotengine/godot/blob/31dfdcb69eb4d98a399243f5fc93b8e14b2e808a/scene/gui/shortcut.cpp#L58

Reply:
Neither are translated - so no worries.

Reply:
Seems to never be exposed to the editor, so it makes sense.